.eslintrc.json

```diff
@@ -8,6 +8,8 @@
       "components": ["Link"],
       "specialLink": ["to"]
     }],
-    "react/prop-types": [0]
+    "react/prop-types": [0],
+    "jsx-a11y/no-noninteractive-element-interactions": [0],
```

Comment:
Alright, I do understand why `jsx-a11y/click-events-have-key-events` is needed. But why do we need this one? There's no rationale in the commit message.

Reply:
In our case we sometimes use click events on li, div, ul, etc elements which are not called `non-interactive-elements`. Since we can't move this events to other elements and we have this eslint rule enabled we will have an error. But with this rule disabled its possible for us to use such events